src/platform/qt/MemoryView.cpp

```diff
@@ -200,13 +201,10 @@ MemoryView::MemoryView(CorePointerSource* controller, QWidget* parent)
 	connect(m_ui.hexfield, &MemoryModel::selectionChanged, &m_malModel, &MemoryAccessLogModel::updateSelection);
 	connect(m_ui.segments, static_cast<void (QSpinBox::*)(int)>(&QSpinBox::valueChanged),
 	        &m_malModel, &MemoryAccessLogModel::setSegment);
-	connect(m_ui.accessLoggerButton, &QAbstractButton::clicked, this, [this]() {
+	connect(m_ui.accessLoggerButton, &QAbstractButton::clicked, PopupManager<MemoryAccessLogView>().constructWith([this] {
 		std::weak_ptr<MemoryAccessLogController> controller = m_controller->memoryAccessLogController();
-		MemoryAccessLogView* view = new MemoryAccessLogView(controller);
-		connect(m_controller.get(), &CoreController::stopping, view, &QWidget::close);
-		view->setAttribute(Qt::WA_DeleteOnClose);
-		view->show();
-	});
+		return new MemoryAccessLogView(controller);
+	}));
```

Comment:
Unlike my previous implementation, I opted not to unify the two places where `MemoryAccessLogView`. This means it's possible to get two copies of the window open (one from the menu, one from the button in `MemoryView`) but since you already did the work to make sure they stay sufficiently in sync I didn't consider it to be worth the effort right now.

(A later PR (a replacement for https://github.com/ahigerd/mgba/pull/2 unless I decide to reuse the same branch) will be refactoring how the `Action`s are managed to separate them from the `Window` class. At that point it might be practical for `MemoryView` to be given a pointer to the action collection so it can use _that_ `PopupManager<MemoryAccessLogView>` instead of having one of its own.)